utils/timezone.tsx

```diff
@@ -11,20 +11,20 @@ export function getBrowserUtcOffset() {
     return moment().utcOffset();
```

Comment:
also this can be a `ts` file instead of `tsx`

Reply:
oops, sorry about that.